docker/test/integration/features/opcua.feature

```diff
@@ -0,0 +1,153 @@
+Feature: Putting and fetching data to OPC UA server
+  In order to send and fetch data from an OPC UA server
+  As a user of MiNiFi
+  I need to have PutOPCProcessor and FetchOPCProcessor
+
+  Background:
+    Given the content of "/tmp/output" is monitored
+
+  Scenario Outline: Create and fetch data from an OPC UA node
+    Given a GetFile processor with the "Input Directory" property set to "/tmp/input" in the "create-opc-ua-node" flow
+    And a file with the content "<Value>" is present in "/tmp/input"
+    And a PutOPCProcessor processor in the "create-opc-ua-node" flow
+    And a FetchOPCProcessor processor in the "fetch-opc-ua-node" flow
+    And a PutFile processor with the "Directory" property set to "/tmp/output" in the "fetch-opc-ua-node" flow
+    And these processor properties are set:
+      | processor name    | property name               | property value                  |
+      | PutOPCProcessor   | Parent node ID              | 85                              |
+      | PutOPCProcessor   | Parent node ID type         | Int                             |
+      | PutOPCProcessor   | Target node ID              | 9999                            |
+      | PutOPCProcessor   | Target node ID type         | Int                             |
+      | PutOPCProcessor   | Target node namespace index | 1                               |
+      | PutOPCProcessor   | Value type                  | <Value Type>                    |
+      | PutOPCProcessor   | OPC server endpoint         | opc.tcp://opcua-server:4840/    |
+      | PutOPCProcessor   | Target node browse name     | testnodename                    |
+      | FetchOPCProcessor | Node ID                     | 9999                            |
+      | FetchOPCProcessor | Node ID type                | Int                             |
+      | FetchOPCProcessor | Namespace index             | 1                               |
+      | FetchOPCProcessor | OPC server endpoint         | opc.tcp://opcua-server:4840/    |
+      | FetchOPCProcessor | Max depth                   | 1                               |
+
+    And the "success" relationship of the GetFile processor is connected to the PutOPCProcessor
+    And the "success" relationship of the FetchOPCProcessor processor is connected to the PutFile
+
+    And an OPC UA server is set up
+
+    When all instances start up
+    Then at least one flowfile with the content "<Value>" is placed in the monitored directory in less than 60 seconds
+
+  Examples: Topic names and formats to test
+    | Value Type   | Value   |
+    | String       | Test    |
+    | UInt32       | 42      |
+    | Double       | 123.321 |
+    | Boolean      | False   |
+
+  Scenario Outline: Update and fetch data from an OPC UA node
```

Comment:
The node ids are set to specific ids that are already present in the demo application server. In the previous test we created a new node id instead of updating an existing one.

Reply:
Sorry, I don't understand -- are the node IDs 51034, 51001 etc defined in the `lordgamez/open62541` image?

Reply:
Yes, the test server application which is started from the container image already defines a list of demo values for each node type of OPC UA.

Reply:
Can you add a comment about this?  Currently this is not visible anywhere in the code changes of the PR.

Reply:
Sure, added a comment in 1663b821d5a73afb19902a47a8fc7620d9f7a380